### ISSUE TITLE
Text changes for 2023 season launch

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3649]
Running list of needed small text changes:

- [ ] In the side-bar to do list on the student dashboard, the mouseover text for "Pitch your Project" should read "This feature will open later in the season if your local chapter is hosting an event."

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/3649?planning-poker) with Zenhub @shaunxp20 @viviancan

**Comment by shaunxp20** [comment]
> @dboyer @shaunxp20 Text changes from the last checkbox are in PR [3668](https://github.com/Iridescent-CM/technovation-app/pull/3668)
> 
> I ended up tagging Shaun for a quick review because the changes were a little heavier than initially expected.

These text changes are on QA. 👍

**Comment by shaunxp20** [comment]
> In the side-bar to do list on the student dashboard, the mouseover text for "Pitch your Project" should read "This feature will open later in the season if your local chapter is hosting an event."

> Add text near parent email in registration process saying "A parental consent form will be sent to this email address." (SA)

These two changes are on QA now.

**Comment by shaunxp20** [comment]
> It looks like the mouseover text for the "Pitch your Project" entry is also appearing for Scores and Certificates."

Good catch on this, the mouseover tooltip was being shared/used by both of those^. I made the tooltips separate. It's on QA now. 😄

**Comment by dboyer** [comment]
Confirmed the updated text is correct on QA.

**Comment by dboyer** [comment]
Confirmed that the mouseover text change is on production.
